gipc/gipc.py

```diff
@@ -57,6 +58,19 @@
 log = logging.getLogger("gipc")
 
 
+# Python 3.8 on MacOS changed the default subprocess method from 
+# fork to spawn. Spawn does not copy any state from the
+# old process. So we should force forking.
+"""
+try:
+    multiprocessing.set_start_method('fork')
+except RuntimeException:
+    # Raises a runtime exception if already set.
+    # If not set to fork, raice an exception telling the user so
+    if multiprocessing.get_start_method() != 'fork':
+        raise Exception("gipc requires multiprocessing start method 'fork'")
+"""
+
```

Comment:
Commented this out, and fixed tests for spawn() on OSX.

We can remove it entirely at this point, and I can set mode to fork in application code.

`monkey.patch_all()` sets start method to forking automatically, btw.